### ISSUE TITLE
Using TravisCI for S3 Website

### ISSUE THREAD

**Issue by erikNeville** [ActoKids/AD440_W19_CloudPracticum#40]

**Comment by yahya09206** [comment]
Estimated Time: 4 hours
Actual Time: 1.5hrs

After my initial research of uploading travisci build to s3, it is possible and going through a few travisci articles it seems relatively simple to implement. Will update this with steps to set up.  Moving initial react app to my groups repo instead of starting everything from scratch.

All checks are passing 

Steps to set up (credit: https://medium.com/@michal.frystacky/static-site-github-to-s3-770953a90f67)
- To get started, login to your AWS account (or create one).
-Create S3 bucket
-Git clone the repo onto your machine
-Once your repository is created login into TravisCI and make sure the repository is toggled to on

Travis — Yaml Config
- According to the Travis docs, the bare minimal configuration we need to add to our .travis.yml to deploy to S3 is:
- deploy:
  provider: s3
  access_key_id: "YOUR AWS ACCESS KEY"
  secret_access_key: "YOUR AWS SECRET KEY"
  bucket: "S3 Bucket"
- We replace the access key and secret key with the ones we got from our newly created user’s credentials.csv file and the bucket should match our S3 created bucket name.

Convert to nodejs
- Add following to top of yml file
 -language: node_js
node_js:
  - node
-make package.json file(default values are fine'
-npm init
-replace 'exit 1 with exit 0' on test line

Build link: https://travis-ci.org/ActoKids/web-ui
pull request link: https://github.com/ActoKids/web-ui/pull/10
wiki link: https://github.com/ActoKids/AD440_W19_CloudPracticum/wiki/Travis-CI-with-S3
tested by bmenaz & told me to change the bucket name in the .yml file(fixed)
I tested it by going to the Travisci website & making sure the build was passing every time I made a pull request.
